core/camel-core-model/src/main/java/org/apache/camel/model/ResumableDefinition.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.model;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlTransient;
+
+import org.apache.camel.ResumeStrategy;
+
+public class ResumableDefinition extends OutputExpressionNode {
+    @XmlAttribute(required = true)
+    private String resumeStrategyRef;
```

Comment:
Rename to `resumeStrategy`

Reply:
I am not entirely sure I understand what your are pointing here as we already have a resumeStrategy member, but I'll do some change and I'll get back to it.